Project-1/README.md

```diff
@@ -63,60 +68,256 @@ T	T	F	T	T
   ```(a)  B that says that x is a brother of y
   (b)  A that says that x is an aunt of y
   (c)  C that says that x and y are cousins 
   (d)  O that says that x is an only child
   (e)  T that says that x has exactly two brothers 
-```
 
-5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+  ⊔ ⊓ ⊧ ⊭ ⊦ ⊬ ⊏ ⊐ ⊑ ⊒ C ¬ ≡ ≠ ≥ ≤
+  parent_of
+  m
+
+  (a)  B that says that x is a brother of y
+  (a)  B ≡ M ⊓ ∃parentOf^-. (∃parentOf≥2)
+  (a) assuming gender binary determined by chromosomes 
+
+  (b)  A that says that x is an aunt of y
+  (b)  B ≡ M ⊓ ∃parentOf^-. (∃parentOf.(≥22parentOf.(parentOfM ⊔ ¬M) ⊓ ∃parentOf≥2)
+  (b) assuming gender binary determined by chromosomes 
+
+  (c)  C that says that x and y are cousins 
+  (c)  C ≡ ∃parentOf^-.(∃parentOf^-.(≥22parentOf.(parentOf M ⊔ ¬M))
+
+  (d)  O that says that x is an only child
+  (d) $O ≡ ∃parentOf^-.(∃parentOf.≤1)
+
+  (e)  T that says that x has exactly two brothers 
+  (e)  T ≡ M ⊓ ∃parentOf^-.(≤3∃parentOf. M) ⊔ (¬M ⊓ ∃parentOf^-.(≤2∃parentOf^-.M ⊓ ≥ 2∃parentOf^-.M))
+
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both.
+
+ TBox = {Brother ⊆ Male,		(TBox.1)
+	Aunt ⊆ ~Male}		(TBox.2)
+
+Donnie = {Frank: Brother ∩ Male		(Axiom.1)
+	Remi: Brother ∩ Male		(Axiom.2)
+	Mary: Aunt			(Axiom.3)
+	(Donnie, Frank): parent_of	(Axiom.4)
+	(Donnie, Remi): parent_of}		(Axiom.5)
+	
+ΔI = {w, x, y, z},
+William  = w
+Xaivar = x
+Eunice = y
+Zane = z
+Male = {w, x}
+Brother = {w, x}
+Aunt = {y}
+Parent_of = {(z,w), (z,x)}
 
 6. Explain the difference - using natural language - between the first-order prefixes:
-  ```(a) ∃x∀y and ∀x∃y
+ ```(a) ∃x∀y and ∀x∃y
   (b) ∃x∀y∃z and ∀x∃y∀z 
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
-```
-	
+
+```(a) ∃x∀y says something like there is some x such that for all y
+(a) ∀x∃y says for all x there is some y 
+(b) ∃x∀y∃z says there is some x such that for all y there is some z such that 
+(b) ∀x∃y∀z says for all x there is come y such that for all x
+(c) ∀x∃y∀z∃w says for all x there is some y such that for all z there is some w
+(c) ∃x∀y∃z∀w says there is some x such that for all y there exists some z such that for all w
+
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+```mermaid
+graph TD;
+0 --> 0;
+0 --> 1;
+1 --> 1;
 ```
+
```

Comment:
Not sure whether you were linking an image, in which case I cannot see it.
In any case, I am not sure if I can understand which of the two statements would become false in your model. Would you point it out to me so that I can properly assess your answer?

Reply:
That makes sense because you use directed arrow to show 1 is not R-related to 0. Congrats to using Mermaid!